qlib/SqlUtil.qm

```diff
@@ -2765,6 +2774,69 @@ sql: select 1 as as_int,1.2 as as_float,3.2 as as_numeric,to_timestamp('20150421
     public hash sub cop_length(any column) {
         return make_cop(COP_LENGTH, column);
     }
+
+    #! Truncates a date column or value regarding the given mask. The resulting value remains Qore::date (no conversion to eg. string)
+    /** @par Example:
+        @code
```

Comment:
please use `@code{.py}` or the comments won't be highlighted - also please use `%new-style` in all examples (no `my`, no `$`s) for consistency's sake